SpatialGDK/Source/SpatialGDKTests/Private/LocalDeploymentManager/LocalDeploymentManagerTest.cpp

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "TestDefinitions.h"
+
+#include "LocalDeploymentManager.h"
+#include "SpatialGDKDefaultLaunchConfigGenerator.h"
+#include "SpatialGDKDefaultWorkerJsonGenerator.h"
+#include "SpatialGDKEditorSettings.h"
+
+#include "CoreMinimal.h"
+
+#define LOCALDEPLOYMENT_TEST(TestName) \
+	TEST(Services, LocalDeployment, TestName)
+
+namespace
+{
+	// TODO: UNR-1969 - Prepare LocalDeployment in CI pipeline
+	static const double MAX_WAIT_TIME_FOR_LOCAL_DEPLOYMENT_OPERATION = 10.0;
+
+	// TODO: UNR-1964 - Move EDeploymentState enum to LocalDeploymentManager
+	enum class EDeploymentState { IsRunning, IsNotRunning };
+
+	static const FName AutomationWorkerType = TEXT("AutomationWorker");
```

Comment:
These are in an anonymous namespace, they don't need to be static.

Reply:
Addressed.